src/raft.cpp

```diff
@@ -228,5 +225,17 @@ Raft::LayerType Raft::getLayerType(LayerIndex layer_index)
     }
 }
 
+size_t Raft::getLayersAmount(const std::string& extruder_nr, const std::string& layers_nr)
```

Comment:
I'm not a big fan if the variable naming here
```suggestion
size_t Raft::getLayersAmount(const std::string& target_extruder_nr, const std::string& target_raft_section)
```